public/stylesheets/site/2.0/21-userstuff.css

```diff
@@ -22,6 +23,10 @@
   margin: auto;
 }
 
+.faq .userstuff dl {
```

Comment:
Could you put the new style rules under the `/* contexts */` comment, at the bottom? That's where we put overrides that should only affect `.userstuff` styles in certain contexts, e.g., bookmarks or FAQs.

Reply:
Fixed!